learning_assistant/views.py

```diff
@@ -149,3 +154,69 @@ def get(self, request, course_run_id):
         }
 
         return Response(status=http_status.HTTP_200_OK, data=data)
+
+
+class LearningAssistantMessageHistoryView(APIView):
+    """
+    View to retrieve the message history for user in a course.
+
+    This endpoint returns the message history stored in the LearningAssistantMessage table in a course
+    represented by the course_key, which is provided in the URL.
+
+    Accepts: [GET]
+
+    Path: /learning_assistant/v1/course_id/{course_key}/history
+
+    Parameters:
+        * course_key: the ID of the course
+
+    Responses:
+        * 200: OK
+        * 400: Malformed Request - Course ID is not a valid course ID.
+    """
+
+    authentication_classes = (SessionAuthentication, JwtAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, course_run_id):
+        """
+        Given a course run ID, retrieve the message history for the corresponding user.
+
+        The response will be in the following format.
+
+            {'enabled': <bool>}
```

Comment:
NIT: this isn't the correct response for this view

Reply:
Not quite sure how to format this here: https://github.com/edx/learning-assistant/pull/119/files#diff-bf828b0d52eaac58c68450b8851259f392b937ae1a26b7cc74749fe9afe970beR183

Reply:
It should be a list of messages, so something like [{'role': 'assistant', 'content': 'something'}]